### ISSUE TITLE
Assume `General` category in `settings` command

### ISSUE THREAD

**Issue by lionkor** [BeamMP/BeamMP-Server#316]
Typing `settings set General MaxPlayers 34` is more common than setting `Misc` or other settings, and thus we should provide overloads for `settings set` which only take a setting name, and either:

- bail out + refuse when it's ambiguous, or
- assume `General` and roll with it.

Related to #295

**Comment by jimkoen** [comment]
I'd rather we drop namespaces completely (so far we only have two values for the Misc namespace) or add a mandatory delimiter with auto-completion, like `general::max_players`.

**Comment by lionkor** [comment]
We need them for later plugin settings support.

Autocomplete should be trivial to add, lets do that

**Comment by Neptnium** [comment]
@lionkor I feel like putting every plugin in the main config file would be a bad idea.
It could quickly become a huge mess, I personally have my plugins create a shared "configs" folder at the root of the server files, and every plugin create its own .TOML config file with the name of the plugin. (Kinda like minecraft even tho minecraft plugin's config are in the same folder as plugins, but its not possible in beammp as every folder is treated like a plugin in the servers resource folder)

Putting every plugin in the main would have issues when removing a plugin from the server and if plugins have big configs (My next plugin has like 30+ config settings, would be a mess if there are multiple like that)

**Comment by lionkor** [comment]
Either way I'd like to be able to query and change settings via the server cli

**Comment by Neptnium** [comment]
Yeah, I agree, would be a good idea, but just the fact of having every plugin's config in the same file would be a bad idea for me, Idk if you understand my opinion but yeah a cli command to change settings would be good!
